django-datadog/middleware/request_handler.py

```diff
@@ -38,12 +44,15 @@ def process_response(self, request, response):
         # Calculate request time and submit to Datadog
         request_time = time.time() - getattr(request, self.DD_TIMING_ATTRIBUTE)
         tags = self._get_metric_tags(request)
-        dog_stats_api.histogram(self.timing_metric, request_time, tags=tags)
+        self.stats.histogram(self.timing_metric, request_time, tags=tags)
 
         return response
 
     def process_exception(self, request, exception):
         """ Captures Django view exceptions as Datadog events """
+        if isinstance(exception, Http404):
+            # Don't report 404 not found
+            return
```

Comment:
Error 404 may be useful information for quite a few use cases. This should at least be configurable, ideally report everything by default.